mr/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergOutputCommitter.java

```diff
@@ -223,6 +225,9 @@ private JobConf jobConf(Table table, int taskNum) {
     conf.setNumMapTasks(taskNum);
     conf.setNumReduceTasks(0);
     conf.set(HiveConf.ConfVars.HIVEQUERYID.varname, QUERY_ID);
+    Map<String, String> outputs = Maps.newHashMap();
+    outputs.put(table.location(), table.name());
```

Comment:
The key and value look reversed here? This is name to location in the storage handler.

Reply:
Test only issue. The test only run for HadoopTables where the `location` and the `name` is the same. Fixed this.

Also we have tests for different `Catalog` implementations where these are different, so the production code is ok.